pom.xml

```diff
@@ -528,10 +528,14 @@ flexible messaging model and an intuitive client API.</description>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.19.1</version>
+        <version>2.20</version>
         <configuration>
           <argLine> -Xmx2G -XX:MaxDirectMemorySize=8G
             -Dio.netty.leakDetectionLevel=advanced</argLine>
+          <redirectTestOutputToFile>true</redirectTestOutputToFile>
+          <reuseForks>false</reuseForks>
+          <forkCount>1</forkCount>
+          <shutdown>kill</shutdown>
```

Comment:
Yes, I'm still playing with it. The idea is that if we run with individual forks for each test class, there will be less interferences between different tests.

Also, we can parallelize and run tests, reducing the overall time from 22min to 6min.